source/extensions/filters/network/kafka/serialization.h

```diff
@@ -224,6 +224,72 @@ class VarUInt32Deserializer : public Deserializer<uint32_t> {
   bool ready_ = false;
 };
 
+class VarInt32Deserializer : public Deserializer<int32_t> {
```

Comment:
should we add class comments like the other classes?

Reply:
Great idea!